src/hid-sensor.cpp

```diff
@@ -145,7 +145,7 @@ void hid_sensor::close()
         std::lock_guard< std::mutex > lock( _configure_lock );
         _configured_profiles.clear();
         _is_configured_stream.clear();
-        _is_configured_stream.resize( RS2_STREAM_COUNT );
+        _is_configured_stream.assign(RS2_STREAM_COUNT, false);
```

Comment:
Please run some sanity on IMU
Run Each one
Run both
Run each one again

Reply:
checked and didn't find any errors because of this change